Fase-1/API/self-order-management/src/main/kotlin/com/fiap/selfordermanagement/application/ports/incoming/LoadOrderUseCase.kt

```diff
@@ -0,0 +1,26 @@
+package com.fiap.selfordermanagement.application.ports.incoming
+
+import com.fiap.selfordermanagement.application.domain.entities.Order
+import com.fiap.selfordermanagement.application.domain.valueobjects.OrderStatus
+
+interface LoadOrderUseCase {
+    fun getByOrderNumber(orderNumber: Long): Order
```

Comment:
Estou usando a seguinte convenção:

- `get` retorna não nullable (lançando exception se o item não for encontrado)
- `find` retorna nullable (um item, ou uma lista, que pode ser vazia)